### DISCUSSION TITLE
Algolia for documentation search

### DISCUSSION THREAD

**Discussion by cliffordfajardo** [xyflow/xyflow#1776] [Ideas]
I think this would enhance the developer experience for the docs - its also free
- https://www.algolia.com/for-open-source/

Live examples
- [Reactjs docs search bar](https://reactjs.org/docs/getting-started.html)
- [Eslint docs search bar](https://eslint.org/)

**Comment by moklick** [comment]
Thanks for the reminder! We already have a branch for this ;) https://github.com/wbkd/react-flow-docs/tree/algolia but it needs some styling.
